src/AutorestSwift/View Models/ObjectViewModel.swift

```diff
@@ -96,7 +96,7 @@ struct ObjectViewModel {
             // the source of flattened parameters should not be included in the view model
 
             // FIXME: This assumption no longer holds for storage and should be revisited
-            assert(property.originalParameter.count <= 1, "Expected, at most, one original parameter.")
+            // assert(property.originalParameter.count <= 1, "Expected, at most, one original parameter.")
```

Comment:
Does this assert fail for paging?

Reply:
Yes. I see when generating code for paging.json

```
WARNING (PreCheck/CheckDuplicateSchemas): Checking for duplicate schemas, this could take a (long) while.  Run with --verbose for more detail.
Assertion failed: Expected, at most, one original parameter.: file AutorestSwift/ObjectViewModel.swift, line 175
Process() cancelled due to failure
```

Reply:
In that case, what are the other parameters? Sometimes they are duplicates of `first` but, at least in storage, there were multiple, different values, in which case we would need to figure out what the right answer should be.

Reply:
They look duplicate to me. But are 'apiVersion' in the case of paging.json